### ISSUE TITLE
Highlight overlapping locations

### ISSUE THREAD

**Issue by dpalomino** [Cadasta/cadasta-platform#1580]
**User Story 1**
As a project user, I'd like to see highlighted on the map overview the neighbour locations (polygons by definition) that are overlapping. 

Additional info: 
- We will not consider locations that are not polygons (polylines or points)
- The overlapped area should be displayed in a different color

**User Story 2**
As a project user, I'd like to search or filter the locations that are overlapping, so I can correct the geometries by snapping to the adjacent locations. 

Additional info: 
- This US would be implemented in a second stage

**Comment by wonderchook** [comment]
@dpalomino I'm thinking we should separate this out and link the two together. Would you see Story 2 be the user is searching an entire large project for all the locations that are overlapping? If that is the case then the technical approach would likely be different than story 1.

**Comment by dpalomino** [comment]
Good point @wonderchook, thanks. Yes exactly, user story 2 is for the user to filter all the locations that are overlapping in a large project, so the user can focus on "fixing" the geometries or raising the boundary conflict. 

I will separate in two different issues now.
